include/silo/storage/column/column.h

```diff
@@ -0,0 +1,16 @@
+#ifndef SILO_COLUMN_H
+#define SILO_COLUMN_H
+
+#include <cstddef>
+#include <cstdint>
+
+namespace silo::storage::column {
+
+class Column {
+  public:
+   virtual std::string getAsString(std::size_t idx) const = 0;
```

Comment:
C++ has too many int types... Should we use `size_t` more consistently? I think we should agree on something^^